src/pages/my-hacktober-experience/hacktober.txt

```diff
@@ -0,0 +1,52 @@
+title:-
+MY EXPERIENCE WITH THE HACKTOBERFEST.
+date:"2019-10-28"
+time: "01:40:00"
```

Comment:
This is somewhat syntactically wrong. Have a look here - https://raw.githubusercontent.com/osdc/blog/master/src/pages/custom-ubuntu-live-cd/index.md

Reply:
Use just title and date. Like this -